MFVNeutralino/test/TrackMoverHighEta/.gitignore

```diff
@@ -0,0 +1,5 @@
+BTagCalibrationStandalone.h
+BTagCalibrationStandalone.cc
+CSVv2_Moriond17_B_F.csv
+CSVv2_Moriond17_B_H.csv
+CSVv2_Moriond17_G_H.csv
```

Comment:
Without looking too closely, I am surprised that btag calibration files are part of a gitignore! The CSVv2 algorithm is old, so we likely should not be using those files anymore regardless (and if they are in the repo anywhere, we should consider removing them). We should track all of the python, C++, and csv/json calibration files that we make use of, and move any others out of the repo, rather than gitignoring them.